README.md

```diff
@@ -184,7 +184,7 @@ Use [Perfherder](https://treeherder.mozilla.org/perf.html#/graphs) directly to d
 Pass a `series` array with each element describing one of your data series. To define a series you need these parameters:
 
 * label - It will be the legend value
-* frameworkId - 1 for Talos, 10 for Raptor and 11 for js-bench.
+* framework - 1 for Talos, 10 for Raptor and 11 for js-bench.
```

Comment:
I've been trying to keep the same nomenclature, however, I might have chosen the internal variable name.
Treeherder's usage of [frameworkId](https://github.com/mozilla/treeherder/search?q=frameworkId&unscoped_q=frameworkId).

Reply:
Treeherder has no standard:  It uses [frameworkId](https://github.com/mozilla/treeherder/search?q=frameworkId&unscoped_q=frameworkId), in code, [framework](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/blob/963d452a7517b5db2c3bf355ea0fc4427ea1220e/src/vendor/perf-goggles.js#L17) in url parameters, and [framework_id](https://github.com/mozilla-frontend-infra/firefox-health-dashboard/pull/336/files#diff-e382eb877fac5502593e9b6b96d9b488R32) in JSON response objects.